tridesclous/catalogueconstructor.py

```diff
@@ -1492,6 +1492,7 @@ def make_catalogue(self):
         #~ self.catalogue['cluster_colors'].update(self.colors)
         
         #params
+        self.catalogue['chunksize'] = self.info['chunksize']
```

Comment:
This addition ensures that the `chunksize` gets saved to file with the catalogue.